dev/doc/release-process.md

```diff
@@ -155,6 +155,8 @@ in time.
 - [ ] Set `is_a_released_version` to `false` in `configure.ml`
   (if you forget about it, you'll be reminded whenever you try to
   backport a PR with a changelog entry).
+- [ ] Add a new link to the ``'versions'`` list of the refman (in
+      ``html_context`` in ``doc/sphinx/conf.py``).
```

Comment:
Actually I would move this under "On the date of the feature freeze" as this can be done as soon as the new branch is created.

Reply:
Done